app/src/js/server/main.ts

```diff
@@ -1,23 +1,25 @@
+import * as child from 'child_process'
 import express, { Application, NextFunction, Request, Response } from 'express'
 import * as formidable from 'express-formidable'
 import { createServer } from 'http'
 import socketio from 'socket.io'
 import 'source-map-support/register'
+import { sprintf } from 'sprintf-js'
```

Comment:
prefer to use template literals in the ES2015.

Reply:
fixed